doc/api/util.md

```diff
@@ -1391,7 +1391,11 @@ added: v10.0.0
 * `value` {any}
 * Returns: {boolean}
 
-Returns `true` if the value is a native `External` value.
+A native `External` value is a special type of object whose
+data is (a) not stored within the JavaScript managed heap
```

Comment:
Micro-nit: I dont' think `(a)` and `(b)` are necessary here.

```suggestion
data is not stored within the JavaScript managed heap
```

Reply:
@Trott , thanks. Fixed it.